app/controllers/webhooks_controller.rb

```diff
@@ -0,0 +1,85 @@
+class WebhooksController < ApplicationController
+  # POST /webhooks
+  def create
+    payload = request.body.read
+    event = nil
+
+    # Verify webhook signature and extract the event
+    # See https://stripe.com/docs/webhooks/signatures for more information.
+    sig_header = request.env['HTTP_STRIPE_SIGNATURE']
+    endpoint_secret = 'whsec_8I8JI2kT6B3RdUR39usVosRMhRvLVrcH'
```

Comment:
we should use ruby secrets store more credentials like these

Reply:
Agreed. This is a test secret so it's nbd right now. But we should pull all of these out and use ruby secrets and heroku config vars to deploy